letsencrypt/configuration.py

```diff
@@ -3,6 +3,7 @@
 import urlparse
 import re
 
+import letsencrypt
```

Comment:
nit: Our normal import order is modules from the standard library, modules from our dependencies, followed by our own modules. With this in mind, I'd put this right before all of the `from letsencrypt...`, preserving the space between the `from acme import challenges` line above it.